tests/NLog.UnitTests/LayoutRenderers/Wrappers/WhenEmptyTests.cs

```diff
@@ -71,10 +71,13 @@ public void CoalesceWithANestedLayout()
         [Fact]
         public void WhenEmpty_MissingInner_ShouldNotThrow()
         {
-            LogManager.ThrowExceptions = true;
-            SimpleLayout l = @"${whenEmpty:whenEmpty=${literal:text=c:\logs\}:inner=${environment:LOG_DIR_XXX}}api.log";
-            var le = LogEventInfo.Create(LogLevel.Info, "logger", "message");
-            Assert.Equal("api.log", l.Render(le));
+            using (new NoThrowNLogExceptions())
+            {
+                SimpleLayout l = @"${whenEmpty:whenEmpty=${literal:text=c\:\\logs\\}}api.log";
```

Comment:
Should remain `SimpleLayout l = @"${whenEmpty:whenEmpty=${literal:text=c:\logs\}:inner=${environment:LOG_DIR_XXX}}api.log";` (To ensure Inner becomes `null`)